modules/ROOT/pages/producing-ign.adoc

```diff
@@ -0,0 +1,64 @@
+:experimental:
+= Fedora CoreOS - Producing an Ignition File
+
+== Scope
+This guide provides easy, step-by-step instructions to produce a basic Ignition file.
+
+Fedora CoreOS (FCOS) Ignition files specify the configuration of FCOS. The process begins with a YAML configuration file. The FCOS Configuration Transpiler (FCCT) converts the human-readable YAML file into machine-readable JSON.
+
+TIP: Once FCOS boots and ingests the Ignition file, the configuration is immutable. Therefore, plan your Ignition file with the configuration that you will need.
+
+== A Simple Example
+This section provides instructions to create a basic Ignition file that produces a user named `fcos_user` that can SSH in with a key.
+
+The overall steps are as follows:
+
+. Write the Fedora CoreOS Configuration (FCC) file in the YAML format.
+. Download and install the Fedora CoreOS Configuration Transpiler (fcct).
+. Use fcct to convert the human-readable YAML FCC file into a machine-readable JSON Ignition file.
+
+=== Prerequisite
+Create the `fcos_user` on your local machine and generate an SSH key for this user. When you SSH into FCOS, you will use the SSH key from the local host.
```

Comment:
We already have a privileged default user called `core`, can we used that instead?

Reply:
Why would you want to do that? It's quite reasonable to assume that people want to load their own usernames and login settings.

Reply:
@lucab I didn't realize that the user named `core` is the default privileged user. I will use that in the context of this simple example.
@Conan-Kudo I will provide more details about adding users when I break down and spell out the fcct specs.

Reply:
@Conan-Kudo I agree with that, but I think that goes to its own page. Something like https://coreos.com/os/docs/latest/adding-users.html.

I think we can leave that out of the scope of this PR. I've noted it at https://github.com/coreos/fedora-coreos-docs/issues/23 for later.